src/Anonymization/Anonymizer/AbstractAnonymizer.php

```diff
@@ -27,8 +27,14 @@ final public function __construct(
         protected string $columnName,
         protected Connection $connection,
         protected Options $options,
+        private ?string $salt = null,
```

Comment:
Why not 
```suggestion
        private string $salt,
```
?

And put the `::generateRandomSalt` method on the `Anonymizator` class ?

Reply:
I'm not sure, anonymizers could be services one day, and I don't see why we should then force the salt to be in the constructor ?

Reply:
makes sens, then maybe we should push it throw options ?

Reply:
If the correct word was "thought" then yes, it'd also make sense, as you prefer I have no strong opinion. Maybe we'd need some kind of common validation callback for options, that would validate all common options ?

Reply:
One of the goal of this class was to not have to update the AbstractAnonymizer constructor signature in the future and as this property will only affect a few anonymizers, I prefer that it not pass the options.

Reply:
And yes, it could be a good idea to had a method `AbstractAnonymizer::validateOptions(Options $options)`  to let each anonymizer validate the options you give to it

Reply:
yep